app/models/ext_management_system.rb

```diff
@@ -745,6 +797,15 @@ def allow_targeted_refresh?
 
   private
 
+  # Child managers went to/from maintenance mode with parent
+  def change_maintenance_for_child_managers
+    if enabled?
+      child_managers.each(&:enable!)
```

Comment:
Shouldn't this be using `pause!` and `resume!`?

Reply:
I wanted to do it, but it was change request by some other reviewers above

Reply:
see `before_` callbacks comment https://github.com/ManageIQ/manageiq/pull/17452#issuecomment-432958273

Reply:
:+1: that explains why it was working

Is there any downside to calling `pause!`/`resume!` though?  Even if the callbacks would fix it why rely on that when we could just as easily call `resume!` directly here?